content/sensu-go/5.17/guides/scale-event-storage.md

```diff
@@ -178,8 +179,86 @@ You may see old events in the web UI or sensuctl output until the etcd datastore
+# make a copy of the current pg_hba.conf
+sudo cp /var/lib/pgsql/data/pg_hba.conf /var/tmp/pg_hba.conf.bak
+# give repl user permissions to replicate from the standby host
+export STANDBY_IP=192.168.52.10
+echo "host replication repl ${STANDYB_IP}/32 md5" | sudo tee -a /var/lib/pgsql/data/pg_hba.conf
+# restart postgresql service to activate pg_hba.conf changes
+sudo systemctl restart postgresql
+{{< /highlight >}}
+
+The streaming replication configuration parameters must be set on the primary Postgres host:
+
+{{< highlight shell >}}
+# make a copy of the postgresql.conf
+sudo cp -a /var/lib/pgsql/data/postgresql.conf /var/lib/pgsql/data/postgresql.conf.bak
+# Append the necessary configuration options
+# To enable read-only queries on a standby server, wal_level must be set to
+# "hot_standby". But you can choose "archive" if you never connect to the
+# server in standby mode.
+echo 'wal_level = hot_standby' | sudo tee -a /var/lib/pgsql/data/postgresql.conf
+
+# Set the maximum number of concurrent connections from the standby servers.
+echo 'max_wal_senders = 5' | sudo tee -a /var/lib/pgsql/data/postgresql.conf
+
+# To prevent the primary server from removing the WAL segments required for
+# the standby server before shipping them, set the minimum number of segments
+# retained in the pg_xlog directory. At least wal_keep_segments should be
+# larger than the number of segments generated between the beginning of
+# online-backup and the startup of streaming replication. If you enable WAL
+# archiving to an archive directory accessible from the standby, this may
+# not be necessary.
+echo 'wal_keep_segments = 32' | sudo tee -a /var/lib/pgsql/data/postgresql.conf
+# restart postgresql service to activate postgresql.conf changes
+sudo systemctl restart postgresql
+{{< /highlight >}}
+
+The standby host must be bootstrapped using the `pg_basebackup` command.
+The process will copy all configuration files from the primary as well as databases.
+
+{{< highlight shell >}}
+# If the standby host has ever run postgres the data directory must be emptied
+sudo systemctl stop postgresql
+sudo mv /var/lib/pgsql/data /var/lib/pgsql/data.bak
+# make the standby data directory
+sudo install -d -o postgres -g postgres -m 0700 /var/lib/pgsql/data
+# bootstrap the standby data directory
+export PRIMARY_IP=192.168.52.11
+sudo -u postgres pg_basebackup -h $PRIMARY_IP -D /var/lib/pgsql/data -P -U repl -R --xlog-method=stream
+Password: 
+30318/30318 kB (100%), 1/1 tablespace
+# Remove configurations that are only for the primary
+sudo sed -r -i.bak '/^(wal_level|max_wal_senders|wal_keep_segments).*/d' /var/lib/pgsql/data/postgresql.conf
+# Enable read-only queries, modify/remove if master wal_level is archive
+echo 'hot_standby = on' | sudo tee -a /var/lib/pgsql/data/postgresql.conf
+# start the postgresql service
+sudo systemcl start postgresql
+{{< /highlight >}}
+
+With this configuration complete, your Sensu events will be replicated to the standby host.
```

Comment:
Should add the psql commands that let you verify replication is happening.

Reply:
Added the commands.